### DISCUSSION TITLE
Amazon SES: `ProvideCredentials` example?

### DISCUSSION THREAD

**Discussion by 4kimov** [awslabs/aws-sdk-rust#962] [Q&A]
This is an example of how to use non-env credentials for `aws_config::SdkConfig`: https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/rust_dev_preview/examples/sdk-config/src/bin/no_env.rs

However, `aws_sdk_sesv2::Client::new()` seems to accept `aws_types::sdk_config::SdkConfig` and I can't get that example to work. Is there a working example anywhere?

**Comment by jdisanti** [comment]
Is it failing due to a missing behavior version? It looks like that example is missing:
```
.behavior_version(aws_sdk_s3::config::BehaviorVersion::latest())
```
In the config part.

**Reply by 4kimov** [reply]
Thanks for the tip. In my case -- the example url I gave above works really well for SQS (without `behavior_version`), but not for SES. I tried adding/removing `behavior_version` to the config for SES, but still no luck. Just cannot get `aws_types::sdk_config::SdkConfig` to play nice with custom credentials params...
